scripts/data_prep/convert_text_to_mds.py

```diff
@@ -20,8 +23,16 @@
 from llmfoundry.data import ConcatTokensDataset
 from llmfoundry.utils.data_prep_utils import (DownloadingIterable,
                                               merge_shard_groups)
+from llmfoundry.utils.exceptions import (InputFolderMissingDataError,
+                                         OutputFolderNotEmptyError)
 
 log = logging.getLogger(__name__)
+mosaicml_logger = None
+if os.environ.get(MOSAICML_PLATFORM_ENV_VAR, 'false').lower(
+) == 'true' and os.environ.get(MOSAICML_ACCESS_TOKEN_ENV_VAR):
+    # Adds mosaicml logger to composer if the run was sent from Mosaic platform, access token is set
+    mosaicml_logger = MosaicMLLogger()
```

Comment:
can we move this to under `if __name__ == '__main__':`, doesn't seem used elsewhere in the file so no need for it to be a global var

Reply:
similar for the convert_delta_to_json file

Reply:
Since this logic is repeated in several places, can we use a helper function instead?